arangod/RestHandler/RestCollectionHandler.cpp

```diff
@@ -341,7 +342,45 @@ void RestCollectionHandler::handleCommandPost() {
     events::CreateCollection(_vocbase.name(), "", TRI_ERROR_BAD_PARAMETER);
     return;
   }
+  auto& cluster = _vocbase.server().getFeature<ClusterFeature>();
+  bool waitForSyncReplication = _request->parsedValue(
+      "waitForSyncReplication", cluster.createWaitsForSyncReplication());
+
+  bool enforceReplicationFactor =
+      _request->parsedValue("enforceReplicationFactor", true);
+
+#if true
```

Comment:
wut?

Reply:
That is to say, just delete the obsolete code underneath if it's not used

Reply:
totally agree, I just keep this in until Jenkins agrees with me that this is all green ;)
Before that I like to be able to switch to old code to see difference in the follow up method call
SO just pretend the else case is gone ;)